arrow/locales.py

```diff
@@ -2311,12 +2312,16 @@ class TurkishLocale(Locale):
         "hours": "{0} saat",
         "day": "bir gün",
         "days": "{0} gün",
+        "week": "hafta",
```

Comment:
Should I write "bir hafta" instead of "hafta"?

"hafta" = "week"
"bir hafta" = "a week"

Reply:
"bir hafta" would be correct.